src/components/Photo.tsx

```diff
@@ -10,12 +10,21 @@ import { PhotoData, BaseProps } from "../data/types"
 interface PhotoProps extends BaseProps {
   photo?: PhotoData
   albumId?: string
+  isBigScreen: boolean
+  setShowHeader: React.Dispatch<React.SetStateAction<boolean>>
 }
 
-function Photo({ photo, albumId, isLoading }: PhotoProps) {
+function Photo({ photo, albumId, isLoading, isBigScreen, setShowHeader }: PhotoProps) {
   const { isOpen, toggle } = useContext(DrawerContext)
   const [touchPosition, setTouchPosition] = useState(null)
+  const [deferredPhoto, setDeferredPhoto] = useState("")
+  const deferredQuery = useDeferredValue(deferredPhoto)
   const navigate = useNavigate()
+  const baseUrl = process.env.REACT_APP_API_BASE_URL
```

Comment:
I don't like using API URLs in the components, ideally everything would be abstracted away in the *data* module.

It's not a big deal as I think I will include the link to the photo as part of the Photo Detail API response, so we can refactor it later.